src/geometry.F90

```diff
@@ -598,8 +599,9 @@ subroutine distance_to_boundary(p, dist, surface_crossed, lattice_translation, &
     real(8) :: d_lat              ! distance to lattice boundary
     real(8) :: d_surf             ! distance to surface
     real(8) :: x0,y0,z0           ! coefficients for surface
+    real(8) :: xyz_cross(3)
```

Comment:
if this variable can be explained succinctly in a few-word comment, that would be nice